### PR TITLE
JAMES-3145 logging org.apache.james.metrics at DEBUG level now logs m…

### PR DESCRIPTION
**Pull request by mbaechler**
…etrics with a 10 seconds interval

	It allows to have some insights on how the service is performing without
	the requirement of having a full metrics infrastructure

### CODE REVIEW

**Comment by chibenwa** [comment]
test this please

**Comment by trantienduchn** [comment]
no log output, test this please

**Comment by mbaechler** [comment]
```

[e2eb1bb2a8cb8157e22a60b602a1d9bbbf745f80] [ERROR] Errors: 
[e2eb1bb2a8cb8157e22a60b602a1d9bbbf745f80] [ERROR]   JamesDeploymentValidationTest>DeploymentValidation.validateDeployment:81 » ConditionTimeout
```

test this please

**Comment by mbaechler** [comment]
Documentation preview:

Screenshot_2020-04-22 Apache James Project – Apache James Server 3 - Metrics
Screenshot_2020-04-22 Apache James Project – Apache James Server 3 - Monitor via Logging

**Comment by mbaechler** [comment]
rebased on master

**Comment by mbaechler** [comment]
```
Could not transfer artifact com.github.fge:throwing-lambdas:jar:0.5.0 from/to central (https://repo.maven.apache.org/maven2): /root/.m2/repository/com/github/fge/throwing-lambdas/0.5.0/throwing-lambdas-0.5.0.jar.part (No such file or directory) 
```

test this please

File: src/site/xdoc/server/monitor-logging.xml [hunk 1]
```diff
@@ -26,18 +26,38 @@
...
+    As most server software, James is using logs to report problems, track weird behaviors and audit
+    what happens during run.
+
+    In the Java ecosystem, it's frequent to use FQDN for logger names: it allows to enable logs for a subtree
+    (i.e. org.apache.james.mailbox for logs about mailbox) or log from a single address
+    (i.e. org.apache.james.mailbox.store.search.MessageSearches).
+
+    These addresses can be actual packages or class names but they don't have to.
+
+    Using class names makes loggers configuration brittle as classes can change quite often or being move from
```
Comment (rouazana) [inline src/site/xdoc/server/monitor-logging.xml:39]:
```suggestion
    Using class names makes loggers configuration brittle as classes can change quite often or being moved from
```

File: src/site/xdoc/server/monitor-logging.xml [hunk 2]
```diff
@@ -26,18 +26,38 @@
...
+    one package to another. It's why you should be careful when using a specific logging address.
+
+    However, for some important logging purpose, James define stable addresses that are documented bellow:
```
Comment (rouazana) [inline src/site/xdoc/server/monitor-logging.xml:42]:
```suggestion
    However, for some important logging purpose, James defines stable addresses that are documented bellow:
```

File: src/site/xdoc/server/monitor-logging.xml [hunk 3]
```diff
@@ -26,18 +26,38 @@
...
-       logging level, per component.</p>
-       
-    <p>You can change the logging level while the server is running via JMX 
-       (under org.apache.james/container).</p>
-       
-    <p>All log files are written in the ./log folder. The ./log/james-sever.log gathers all the other logs.
-      Log file rotation is also available.</p>
-    
+
+    As most server software, James is using logs to report problems, track weird behaviors and audit
```
Comment (Arsnael) [inline src/site/xdoc/server/monitor-logging.xml:30]: s/software/softwares

**Comment by mbaechler** [comment]
```

[9d512fefeeea86da761adae7870cbdf4abef57d0] [ERROR] Errors: 
[9d512fefeeea86da761adae7870cbdf4abef57d0] [ERROR]   MemorySpamAssassinContractTest.spamShouldBeDeliveredInSpamMailboxOrInboxWhenMultipleRecipientsConfigurations » ConditionTimeout
```

File: src/site/xdoc/server/monitor-logging.xml (hunk 3 above)
Comment (mbaechler) [inline src/site/xdoc/server/monitor-logging.xml:30]:
```suggestion
    As most server softwares, James is using logs to report problems, track weird behaviors and audit
```

**Comment by mbaechler** [comment]
Merged

**Pull request closed without merging**